documentation/asciidoc/computers/raspberry-pi/power-supplies.adoc

```diff
@@ -91,8 +96,9 @@ The specific power requirements of each model are shown below.
 | Raspberry Pi 3 Model B+ | 2.5A | 1.2A | 500mA 
 | Raspberry Pi 4 Model B | 3.0A | 1.2A | 600mA 
 | Raspberry Pi 400       | 3.0A | 1.2A | 800mA 
-| Raspberry Pi Zero W/WH | 1.2A | Limited by PSU, board, and connector ratings only.| 150mA 
 | Raspberry Pi Zero | 1.2A | Limited by PSU, board, and connector ratings only | 100mA 
+| Raspberry Pi Zero W/WH | 1.2A | Limited by PSU, board, and connector ratings only.| 150mA 
+| Raspberry Pi Zero 2 W | 2A | Limited by PSU, board, and connector ratings only | 350mA
```

Comment:
Is this basically a repetition of the info in the earlier table?? Should one of the tables be removed @aallan ?

Reply:
Maybe. I'll merge this, and then look at the thing in-situ again when the changes are made I think.